libraries/SRV_Channel/SRV_Channel.h

```diff
@@ -52,7 +52,7 @@ class SRV_Channel {
     typedef enum
     {
         k_GPIO                  = -1,           ///< used as GPIO pin (input or output)
-        k_none                  = 0,            ///< disabled
+        k_none                  = 0,            ///< disabled / PWM by default
```

Comment:
I might change this comment to "///< general use PWM output used by do-set-servo commands and lua scripts"

Reply:
ok

Reply:
fixed